src/client/sandbox/event/simulator.js

```diff
@@ -369,10 +356,38 @@ export default class EventSimulator {
             Object.defineProperty(pointEvent, 'buttons', {
                 get: () => args.buttons
             });
+        }
+        else {
+            pointerArgs.bubbles    = true;
+            pointerArgs.cancelable = true;
+
+            pointEvent = new window.PointerEvent(pointerEventType, pointerArgs);
```

Comment:
I think we should to use native classes and methods. They are rarely overridden, but it happens.